### PR TITLE
add Kernel 6.2 support

### PR DESCRIPTION
**Pull request by simonchen007**
patch6.2 added.
and in some newer linux OS(eg: Linux mint 22) , some path of modules are changed.

### CODE REVIEW

File: prepare-for-current-kernel.sh [hunk 1]
```diff
@@ -7,8 +7,11 @@ then
 elif { echo $VERSION ; echo "5.99" ; } | sort -V -c 2>/dev/null
 then
   PATCHFILE="patch5.8"
-else
+elif { echo $VERSION ; echo "6.0" ; } | sort -V -c 2>/dev/null
```
Comment (Plippo) [inline prepare-for-current-kernel.sh:10]: This actual means "VERSION <= 6.0" -> thus it won't match 6.01 etc...
Comparing against "6.19" would be better so it only uses the new patch from 6.2 onwards

**Review by Plippo — CHANGES_REQUESTED** [review]
Thank you for your effort creating the pull request! Would you mind reviewing the suggested change? Thanks in advance!

File: prepare-for-current-kernel.sh (hunk 1 above)
Comment (simonchen007) [reply prepare-for-current-kernel.sh:10]: so, change the fllowing 
```
elif { echo $VERSION ; echo "6.0" ; } | sort -V -c 2>/dev/null
```
to 
```
elif { echo $VERSION ; echo "6.19" ; } | sort -V -c 2>/dev/null
```
right ?

Comment (Plippo) [reply prepare-for-current-kernel.sh]: Yes exactly

**Review by Plippo — APPROVED** [review]

**Pull request merged**